server/sonar-web/src/main/js/components/shared/drilldown-link.js

```diff
@@ -55,7 +55,9 @@ export class DrilldownLink extends React.PureComponent {
   propsToIssueParams = () => {
     const params = {};
 
-    if (this.props.periodDate) {
+    if (this.props.sinceLeakPeriod) {
```

Comment:
some components are still using the `periodDate` property, for example `overview/main/Coverage`, please change them all